File: RQ1_Prompt_Elements/RQ1_Result/coverage_result/src/main/java/scenario2/id_44.java
package scenario2;


class ChangeBase {
    /**
     * Change numerical base of input number x to base.
     * return string representation after the conversion.
     * base numbers are less than 10.
     */
    public static String changeBase(int x, int base) {
        String result = "";
        while (x > 0) {
            result = (x % base) + result;
            x = x / base;
        }
        return result;
    }

}

File: HumanEvalPython/scenario2/id_44.py


def change_base(x: int, base: int):
    """Change numerical base of input number x to base.
    return string representation after the conversion.
    base numbers are less than 10.
    
    """

    ret = ""
    while x > 0:
        ret = str(x % base) + ret
        x //= base
    return ret
